ShelbyFinancials/ReadMe.md

```diff
@@ -40,28 +40,32 @@ After install, the Shelby Financials Batch to Journal block was added to your Ba
 
 ![](../.screenshots/ShelbyFinancials/BatchToJournal.png)
 
-
-
 ![](../.screenshots/ShelbyFinancials/BatchToJournalProperties.png)
 
+<div style="page-break-after: always;"></div>
+
 | | |
 | --- | ---- |
-| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">1</span> | **Name** Block name |
-| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">2</span> | **Button Text** Customize the text on the export button |
-| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">3</span> | **Close Batch** Control whether the batch gets marked as closed in Rock after export
-
-<div style="page-break-after: always;"></div>
+| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">1</span> | **Button Text** Customize the text on the export button |
+| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">2</span> | **Close Batch** Control whether the batch gets marked as closed in Rock after export |
+| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">3</span> | **GL Account Grouping** Determines if debit and/or credit lines should be grouped and summed by GL account in the export file |
```

Comment:
Add detailed note describing what fields are grouped by for debit grouping, credit grouping, and financial account grouping